### DISCUSSION TITLE
Getting generic 'Error' message when trying to log in as Administrator

### DISCUSSION THREAD

**Discussion by Elliot-12345** [globaleaks/globaleaks-whistleblowing-software#3797] [Q&A]
When trying to log in to the admin portal of our Globaleaks whistleblowing app, at the point of submitting credentials, we are seeing a red pop up that just says 'Error!'. We have seen this behaviour in the past and it has randomly resolved itself, we need a way of permanently resolving this.

Please see below for an example:

![image](https://github.com/globaleaks/GlobaLeaks/assets/117072864/c9e0b7c8-91b7-4e97-b336-1144bfe3756c)

**Comment by Elliot-12345** [comment]
This looks to have been due to IP address whitelisting on the Admin user, preventing the login from a new IP address. Seems to be working now.

**Reply by AcidBit** [reply]
I'm facing the same error here. Could you plz be more specific in the solution? Thank you.
